### ISSUE TITLE
I changed "time.perf_counter_ns()" to "time.perf_counter()"

### ISSUE THREAD

**Issue by dadeke** [ShailChoksi/lichess-bot#268]
Hi guys! 🙋‍♂️

I changed **"time.perf_counter_ns()"** to **"time.perf_counter()"** in "lichess-bot.py" because message:
```
2021-02-28 03:30:47,061: +++ https://lichess.org/2XPSop4j/white Bullet vs dadeke(1500?)
2021-02-28 03:30:47,062: Backing off play_game(...) for 43.2s (AttributeError: module 'time' has no attribute 'perf_counter_ns')
```

Is this a issue?

**Comment by AttackingOrDefending** [comment]
time.perf_counter() counts seconds, while time.perf_counter_ns() nanoseconds. This will make your program to think that you have only spent for example 10 nanoseconds instead of 10 seconds. You can add
```python
time.perf_counter_ns = lambda: time.perf_counter() * 1000000000
```
in the start of lichess-bot.py. You can also solve this error by updating to python 3.7 or later.

**Comment by dadeke** [comment]
> 
> 
> in the start of lichess-bot.py to solve this error. It is also solved by updating to python 3.7 or later.

I upgraded my python.

Thanks. 👍
